templates/admin/jqadm/product/item-characteristic-property.php

```diff
@@ -16,7 +23,7 @@
 	<div class="box">
 		<property-table
 			v-bind:domain="'product'" v-bind:siteid="`<?= $enc->js( $this->site()->siteid() ) ?>`" v-bind:tabindex="`<?= $enc->js( $this->get( 'tabindex' ) ) ?>`"
-			v-bind:types="<?= $enc->attr( $this->get( 'propertyTypes', map() )->col( 'product.property.type.label', 'product.property.type.code' )->all() ) ?>"
```

Comment:
You can get the same much simpler if you add the `(object)` type cast in front of `$this->get()`. No checking if the map is empty is necessary :-)

Reply:
done!